docs/plugins/find-and-install-plugin.md

```diff
@@ -0,0 +1,26 @@
+(find-and-install-plugins)=
+
+# Finding and installing a napari plugin
+
+You can find plugins to use with napari on the [napari hub](https://napari.org),
```

Comment:
should we mention [PyPI](https://pypi.org/search/?q=napari&o=&c=Framework+%3A%3A+napari), since it's ultimately the root source of plugins?  Or is that too much?

Reply:
true.  I don't have a _super_ strong feeling here.  But I just kinda keep coming back to the concept of (or at least, my desire for) napari being a regular member of the broader python ecosystem, and less a completely new entity on the scene that hides pre-existing python conventions behind "the napari way".

Reply:
maybe we could mention something about the hub simply being a "catalog" of pypi plugins?

Reply:
cc @neuromusic @LCObus

Reply:
I hear you @tlambert03 , I do think there's a way to commit to napari being a regular member of broader python ecosystem while making the hub more accessible to less python savvy users (who might not know what "pypi" is)...since this page lives on napari site rather than hub, i see no problem making a reference to what the hub is pulling from (we do this in the FAQs for the hub too)

Reply:
The copy I would suggest for this paragraph:

```
napari plugins are Python packages distributed on the Python Package
Index (PyPI), and annotated with the tag [`Framework ::
napari`](https://pypi.org/search/?q=&o=&c=Framework+%3A%3A+napari).
The [napari hub](https://napari-hub.org) uses this data, together with
additional metadata, to produce a more user friendly way to find napari
plugins.
```

I think we should point users to the hub, but in general, where it's easy, I like to let users "peek behind the curtains" and realise that this isn't magical, it's just nice engineering and building on community efforts. 😊

btw note that the link is currently pointing to napari.org rather than napari-hub.org 😂

Reply:
That sounds good to me!